includes/api/class-wc-rest-order-refunds-controller.php

```diff
@@ -369,121 +369,130 @@ public function get_item_schema() {
-							'type'        => 'string',
-							'context'     => array( 'view', 'edit' ),
-							'readonly'    => true,
-						),
-						'subtotal' => array(
-							'description' => __( 'Line subtotal (before discounts).', 'woocommerce' ),
-							'type'        => 'string',
-							'context'     => array( 'view', 'edit' ),
-						),
-						'subtotal_tax' => array(
-							'description' => __( 'Line subtotal tax (before discounts).', 'woocommerce' ),
-							'type'        => 'string',
-							'context'     => array( 'view', 'edit' ),
-						),
-						'total' => array(
-							'description' => __( 'Line total (after discounts).', 'woocommerce' ),
-							'type'        => 'string',
-							'context'     => array( 'view', 'edit' ),
-						),
-						'total_tax' => array(
-							'description' => __( 'Line total tax (after discounts).', 'woocommerce' ),
-							'type'        => 'string',
-							'context'     => array( 'view', 'edit' ),
-						),
-						'taxes' => array(
-							'description' => __( 'Line taxes.', 'woocommerce' ),
-							'type'        => 'array',
-							'context'     => array( 'view', 'edit' ),
-							'readonly'    => true,
-							'properties'  => array(
-								'id' => array(
-									'description' => __( 'Tax rate ID.', 'woocommerce' ),
-									'type'        => 'integer',
-									'context'     => array( 'view', 'edit' ),
-									'readonly'    => true,
-								),
-								'total' => array(
-									'description' => __( 'Tax total.', 'woocommerce' ),
-									'type'        => 'string',
-									'context'     => array( 'view', 'edit' ),
-									'readonly'    => true,
-								),
-								'subtotal' => array(
-									'description' => __( 'Tax subtotal.', 'woocommerce' ),
-									'type'        => 'string',
-									'context'     => array( 'view', 'edit' ),
-									'readonly'    => true,
-								),
+					'items'       => array(
+						'type'       => 'object',
+						'properties' => array(
+							'id' => array(
+								'description' => __( 'Item ID.', 'woocommerce' ),
+								'type'        => 'integer',
+								'context'     => array( 'view', 'edit' ),
+								'readonly'    => true,
 							),
-						),
-						'meta' => array(
-							'description' => __( 'Line item meta data.', 'woocommerce' ),
-							'type'        => 'array',
-							'context'     => array( 'view', 'edit' ),
-							'readonly'    => true,
-							'properties'  => array(
-								'key' => array(
-									'description' => __( 'Meta key.', 'woocommerce' ),
-									'type'        => 'string',
-									'context'     => array( 'view', 'edit' ),
-									'readonly'    => true,
-								),
-								'label' => array(
-									'description' => __( 'Meta label.', 'woocommerce' ),
-									'type'        => 'string',
-									'context'     => array( 'view', 'edit' ),
-									'readonly'    => true,
+							'name' => array(
+								'description' => __( 'Product name.', 'woocommerce' ),
+								'type'        => 'integer',
```

Comment:
It looks like this was wrong in the original code too, but product name isn't an integer.